2021-08-29 10:28:48,089 INFO    MainThread:42564 [wandb_setup.py:_flush():69] setting env: {}
2021-08-29 10:28:48,089 INFO    MainThread:42564 [wandb_setup.py:_flush():69] setting login settings: {}
2021-08-29 10:28:48,089 INFO    MainThread:42564 [wandb_init.py:_log_setup():342] Logging user logs to /opt/ml/code/image-classification-level1-22/T2037/wandb/run-20210829_102848-2sodsuxt/logs/debug.log
2021-08-29 10:28:48,089 INFO    MainThread:42564 [wandb_init.py:_log_setup():343] Logging internal logs to /opt/ml/code/image-classification-level1-22/T2037/wandb/run-20210829_102848-2sodsuxt/logs/debug-internal.log
2021-08-29 10:28:48,089 INFO    MainThread:42564 [wandb_init.py:init():375] calling init triggers
2021-08-29 10:28:48,089 INFO    MainThread:42564 [wandb_init.py:init():380] wandb.init called with sweep_config: {}
config: {}
2021-08-29 10:28:48,089 INFO    MainThread:42564 [wandb_init.py:init():424] starting backend
2021-08-29 10:28:48,089 INFO    MainThread:42564 [backend.py:_multiprocessing_setup():70] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2021-08-29 10:28:48,101 INFO    MainThread:42564 [backend.py:ensure_launched():135] starting backend process...
2021-08-29 10:28:48,109 INFO    MainThread:42564 [backend.py:ensure_launched():139] started backend process with pid: 42587
2021-08-29 10:28:48,125 INFO    MainThread:42564 [wandb_init.py:init():429] backend started and connected
2021-08-29 10:28:48,126 INFO    MainThread:42564 [wandb_init.py:init():477] updated telemetry
2021-08-29 10:28:48,127 INFO    MainThread:42564 [wandb_init.py:init():500] communicating current version
2021-08-29 10:28:48,872 INFO    MainThread:42564 [wandb_init.py:init():505] got version response upgrade_message: "wandb version 0.12.1 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2021-08-29 10:28:48,872 INFO    MainThread:42564 [wandb_init.py:init():513] communicating run to backend with 30 second timeout
2021-08-29 10:28:49,093 INFO    MainThread:42564 [wandb_init.py:init():540] starting run threads in backend
2021-08-29 10:28:54,097 INFO    MainThread:42564 [wandb_run.py:_console_start():1584] atexit reg
2021-08-29 10:28:54,097 INFO    MainThread:42564 [wandb_run.py:_redirect():1458] redirect: SettingsConsole.REDIRECT
2021-08-29 10:28:54,098 INFO    MainThread:42564 [wandb_run.py:_redirect():1463] Redirecting console.
2021-08-29 10:28:54,102 INFO    MainThread:42564 [wandb_run.py:_redirect():1519] Redirects installed.
2021-08-29 10:28:54,102 INFO    MainThread:42564 [wandb_init.py:init():565] run started, returning control to user process
2021-08-29 10:28:57,307 INFO    MainThread:42564 [wandb_run.py:_atexit_cleanup():1554] got exitcode: 1
2021-08-29 10:28:57,307 INFO    MainThread:42564 [wandb_run.py:_restore():1526] restore
2021-08-29 10:29:00,205 INFO    MainThread:42564 [wandb_run.py:_wait_for_finish():1676] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 947
  total_bytes: 947
}

2021-08-29 10:29:00,502 INFO    MainThread:42564 [wandb_run.py:_wait_for_finish():1676] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 947
  total_bytes: 947
}

2021-08-29 10:29:01,108 INFO    MainThread:42564 [wandb_run.py:_wait_for_finish():1676] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 947
  total_bytes: 6975
}

2021-08-29 10:29:01,210 INFO    MainThread:42564 [wandb_run.py:_wait_for_finish():1676] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 947
  total_bytes: 10869
}

2021-08-29 10:29:01,314 INFO    MainThread:42564 [wandb_run.py:_wait_for_finish():1676] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 947
  total_bytes: 10869
}

2021-08-29 10:29:01,418 INFO    MainThread:42564 [wandb_run.py:_wait_for_finish():1676] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 6653
  total_bytes: 10869
}

2021-08-29 10:29:01,521 INFO    MainThread:42564 [wandb_run.py:_wait_for_finish():1676] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 10869
  total_bytes: 10869
}

2021-08-29 10:29:01,622 INFO    MainThread:42564 [wandb_run.py:_wait_for_finish():1676] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 10869
  total_bytes: 10869
}

2021-08-29 10:29:01,727 INFO    MainThread:42564 [wandb_run.py:_wait_for_finish():1676] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 10869
  total_bytes: 10869
}

2021-08-29 10:29:01,829 INFO    MainThread:42564 [wandb_run.py:_wait_for_finish():1676] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 10869
  total_bytes: 10869
}

2021-08-29 10:29:01,931 INFO    MainThread:42564 [wandb_run.py:_wait_for_finish():1676] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 10869
  total_bytes: 10869
}

2021-08-29 10:29:02,109 INFO    MainThread:42564 [wandb_run.py:_wait_for_finish():1676] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 10869
  total_bytes: 10869
}

2021-08-29 10:29:02,211 INFO    MainThread:42564 [wandb_run.py:_wait_for_finish():1676] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 10869
  total_bytes: 10869
}

2021-08-29 10:29:03,349 INFO    MainThread:42564 [wandb_run.py:_show_files():1898] logging synced files
